GVFS/GVFS.Notifications/VFSForGit.Mac/VFSForGit.UnitTests/VFSStatusBarItemTests.m

```diff
@@ -0,0 +1,56 @@
+#import <XCTest/XCTest.h>
+#import "VFSMockAboutWindowController.h"
+#import "VFSMockProductInfoFetcher.h"
+#import "VFSStatusBarItem.h"
+
+NSString * const ExpectedAboutMenuTitle = @"About VFS For Git";
+
+@interface VFSStatusBarItemTests : XCTestCase
+
+@property (strong) VFSStatusBarItem *statusbarItem;
+@property (strong) VFSMockAboutWindowController *aboutWindowController;
+
+@end
+
+@implementation VFSStatusBarItemTests
+
+- (void)setUp
+{
+    [super setUp];
+    
+    VFSMockProductInfoFetcher *mockProductInfoFetcher = [[VFSMockProductInfoFetcher alloc]
+                                                         initWithGitVersion:@""
+                                                         vfsforgitVersion:@""];
+    
+    self.aboutWindowController = [[VFSMockAboutWindowController alloc]
+                                  initWithProductInfoFetcher:mockProductInfoFetcher];
+    self.statusbarItem = [[VFSStatusBarItem alloc]
+                          initWithAboutWindowController:self.aboutWindowController];
+    
+    [self.statusbarItem load];
+}
+
+- (void)tearDown
+{
+    [super tearDown];
```

Comment:
Do we need a method that only calls the super method? I thought this would be the default behavior...

Reply:
Yeah it is not necessary, sometimes people leave it as a placeholder and other times code reviewers remind them to remove the empty impl.